support-frontend/assets/components/checkoutSupportOnce/checkoutSupportOnceContainer.tsx

```diff
@@ -0,0 +1,41 @@
+import { config } from 'helpers/contributions';
+import { detect, glyph } from 'helpers/internationalisation/currency';
+import { getContributionType } from 'helpers/redux/checkout/product/selectors/productType';
+import { useContributionsSelector } from 'helpers/redux/storeHooks';
+import type { CheckoutSupportOnceProps } from './checkoutSupportOnce';
+
+type SupportOnceContainerProps = {
+	renderSupportOnce: (props: CheckoutSupportOnceProps) => JSX.Element;
+	onSupportOnceContainerClick: () => void;
+};
+
+export function CheckoutSupportOnceContainer({
+	renderSupportOnce,
+	onSupportOnceContainerClick,
+}: SupportOnceContainerProps): JSX.Element | null {
+	const contributionType = useContributionsSelector(getContributionType);
+	const { countryGroupId } = useContributionsSelector(
+		(state) => state.common.internationalisation,
+	);
+
+	const currencyGlyph = glyph(detect(countryGroupId));
+	const minAmount = config[countryGroupId]['ONE_OFF'].min;
+
+	const [title, paragraph] = [
+		`Support us just once`,
+		`We welcome support of any size, any time, whether you choose to give ${
+			currencyGlyph + minAmount.toString()
+		} or much more`,
+	];
+
+	function onSupportOnceClick() {
+		onSupportOnceContainerClick();
+	}
+
+	return renderSupportOnce({
+		contributionType,
+		supportOnceTitle: title,
+		supportOnceParagraph: paragraph,
+		onSupportOnceClick,
+	});
```

Comment:
Could you remove `onSupportOnceClick`? I.e. would the following work?

```suggestion
	return renderSupportOnce({
		contributionType,
		supportOnceTitle: title,
		supportOnceParagraph: paragraph,
		onSupportOnceContainerClick,
	});
```

Reply:
This is how the SupportOnce component indicates to the container its been clicked, its passed up from there to SupporterPlusLanding to notify redux to re-display the single tab.  From this point it acts as 3 frequency tabs (single/monthly/annual) as usual. It's needed I believe.

Reply:
Sorry I see, removed click event, and passed through now as 'onSupportOnceClick'